about.html

```diff
@@ -0,0 +1,162 @@
+      </a>
+    </section>
+    <section class="logo-cont">
+      <h2 class="logo-title">this is the logo presentation</h2>
+      <div class="line"></div>
+      <p class="logo-desc">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+      </p>
+      <a class="logo logo-show">
+        <img alt="Agility contest" class="logo-img-show" src="./assets/images/detailed-logo.jpg" />
+        <h3 class="logo-text">
+          agility <br />
+          contest
+        </h3>
+      </a>
+    </section>
+    <section class="past-events">
+      <h2 class="events-title">See the pase agility contests</h2>
+      <div class="line"></div>
+      <p class="event-subtitle">
+        Take a look at the last two Agility contests which took place in Buenos
+        Aires and in Warsaw
+      </p>
+      <div class="events-pics">
+        <div class="event event1">
+          <div class="bg-event">
+            <p class="event-year">2015</p>
+            <p class="event-detail">Agility contest 2013 in Buenos Aires</p>
+          </div>
+        </div>
+        <div class="event event2">
+          <div class="bg-event">
+            <p class="event-year">2015</p>
+            <p class="event-detail">Agility contest 2013 in Buenos Aires</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <section class="partner">
+        <h2 class="sponsors-title">partner</h2>
+        <div class="line"></div>
+        <div class="sponsors">
+          <img
+            class="sponsor-img"
+            src="./assets/images/mozilla.png"
+            alt="mozilla"
+          />
+          <img
+            class="sponsor-img"
+            src="./assets/images/google.png"
+            alt="google"
+          />
+          <img
+            class="sponsor-img"
+            src="./assets/images/naver.png"
+            alt="naver"
+          />
+          <img
+            class="sponsor-img"
+            src="./assets/images/daumkakao.png"
+            alt="daumkakao"
+          />
+          <img
+            class="sponsor-img"
+            src="./assets/images/airBNB.png"
+            alt="airbnb"
+          />
+        </div>
+      </section>
+      <section class="copyright">
+        <a class="logo footer-logo" href="#">
+          <img alt="Agility contest" class="logo-img" src="./assets/images/detailed-logo.jpg" />
+          <h3 class="logo-text">
+            agility <br />
+            contest
+          </h3>
+        </a>
+        <div class="copy-text">
+          <p class="reservation">
+            2015 Agility Contest Argentina, Some Rights Reserved
+          </p>
+          <p class="copy-claim">
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Pariatur
+            minus rem sed ducimus, eius quis repellat culpa alias, asperiores
+            adipisci similique molestias?
+          </p>
+        </div>
+      </section>
+    </footer>
```

Comment:
- [ ] On desktop view the footer has a dark background and on mobile view, it has a white background. Please make the following changes on both pages.

Reply:
Hello, what changes do I have to make to the footer? I am not understanding this one